gradle/maven.gradle

```diff
@@ -69,15 +69,16 @@ plugins.withId('maven') {
             }
         }
     }
-
-    install {
-        repositories {
-            mavenInstaller {
-                pom.artifactId = "pravega" + project.path.replace(':', '-')
-            }
-        }
-    }
-
-    task publish(dependsOn:'uploadArchives') {
-    }
+//
```

Comment:
Why are these commented out?

Reply:
I have changed the maven plugin to maven-publish plugin. In maven-publish, the  "publish" tasks is instead of Maven Plugin "Upload" tasks. The plugin generates Maven metadata and publishes the generated metadata and the associated artifacts to Maven repositories. The plugin will do the publish task. So I thought there is no need to initiate the task again.